scripts/lib.sh

```diff
@@ -1,5 +1,5 @@
 export GOPATH=${GOPATH:-${HOME}/go}
-readonly kata_arch_sh="${GOPATH}/src/github.com/kata-containers/tests/.ci/kata-arch.sh"
+kata_arch_sh="${GOPATH}/src/github.com/kata-containers/tests/.ci/kata-arch.sh"
```

Comment:
if it's supposed to be readonly, wouldn't it be better to do a:
`[ -z "$kata_arch_sh" ] && readonly kata_arch_sh=...`

Reply:
yeah or probably add a check by exporting a variable and check if was is set if set stop the source  not sure if with an exit or something similar